tests/testthat/test_cluster_MiniBatchKmeans.R

```diff
@@ -0,0 +1,24 @@
+context("cluster_MiniBatchKmeans")
+
+test_that("cluster_MiniBatchKmeans", {
+  requirePackagesOrSkip("ClusterR", default.method = "load")
+
+  parset.list = list(
+    list()
```

Comment:
Can you add another parameter setting to test please?

Reply:
Added one more test setting.